notifications/templates/notifications/view_notifications.html

```diff
@@ -0,0 +1,15 @@
+<!-- notifications/view_notifications.html -->
+{% extends 'events/base.html' %}
+
+{% block content %}
+    <h2>All Notifications</h2>
+    {% for notification in notifications %}
+        <div>
+            <p>{{ notification.message }}</p>
+            <small>{{ notification.timestamp }}</small>
+            {% if not notification.is_read %}
+                <a href="{% url 'notifications:mark-notification-as-read' notification.id %}">Mark as Read</a>
```

Comment:
The code looks correct, but when testing locally, the "mark as read" link doesn't show up even when the message first appears. Could you also take a look at it? Thank you!
![image](https://github.com/gcivil-nyu-org/Wednesday-Fall2023-Team-3/assets/130470692/c3627d6c-0dc7-4357-aac0-88e0efa6857a)